src/lumen/app_task.cpp

```diff
@@ -14,7 +14,7 @@ constexpr auto tag = "app_task";
 
 void app_task(void* /* parameters */)
 {
-    auto activity_context = activity::Context{};
+    auto activity_context = activity::Context{activity::Type::football};
```

Comment:
- [ ] I think it makes sense to keep the activity to be none since football is not complete